src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write code here
+function convertToNum(item, key) {
+  return +item[key].replace(/\$|,/g, '') || undefined;
+}
+
+function getEmployees() {
+  const resultArray = [];
+  const employeeList = document.querySelectorAll('li');
+
+  employeeList.forEach(
+    item => resultArray.push({
+      ...item.dataset, name: item.innerText,
+    })
+  );
+
+  for (const employee of resultArray) {
+    employee.salary = convertToNum(employee, 'salary');
+  }
```

Comment:
and call this method right inside the map, no need to loop the second time for that